root/release_group/ReleaseGroupIndex.js

```diff
@@ -0,0 +1,213 @@
+                    />
+                  </td>
+                ) : null
+            )}
+          </CatalystContext.Consumer>
+          <td>
+            <EntityLink entity={release} />
+          </td>
+          <td>{release.combined_format_name || l('[missing media]')}</td>
+          <td>{release.combined_track_count || '-'}</td>
+          <td>
+            <ReleaseDates events={release.events} />
+          </td>
+          <td>
+            <ReleaseCountries events={release.events} />
+          </td>
+          <td>
+            <ReleaseLabelList labels={release.labels} />
+          </td>
+          <td>
+            <ReleaseCatnoList labels={release.labels} />
+          </td>
+          <td className="barcode-cell">{formatBarcode(release.barcode)}</td>
+          <CatalystContext.Consumer>
+            {($c: CatalystContextT) => (
+              $c.session && $c.session.tport
+                ? <td><TaggerIcon entity={release} /></td>
+                : null
+            )}
+          </CatalystContext.Consumer>
+        </tr>
+      ))}
+    </>
+  );
+}
+
+const ReleaseGroupIndex = ({
+  $c,
+  eligibleForCleanup,
+  pager,
+  mostPopularReview,
+  mostRecentReview,
+  numberOfRevisions,
+  releaseGroup,
+  releases,
+  wikipediaExtract,
+}: Props) => (
+  <ReleaseGroupLayout entity={releaseGroup} page="index">
+    {eligibleForCleanup ? (
+      <p className="cleanup">
+        {l(
+          `This release group has no relationships or releases associated,
+           and will be removed automatically in the next few days. If this
+           is not intended, please add more data to this release group.`,
+        )}
+      </p>
+    ) : null
+    }
+    <Annotation
+      annotation={releaseGroup.latest_annotation}
+      collapse
+      entity={releaseGroup}
+      numberOfRevisions={numberOfRevisions}
+    />
+    <WikipediaExtract
+      cachedWikipediaExtract={wikipediaExtract || null}
+      entity={releaseGroup}
+    />
+    {releases.length ? (
+      <>
+        {releaseGroup.typeName ? <h2>{l(releaseGroup.typeName)}</h2> : null}
+        <form action="/release/merge_queue" method="post">
+          <PaginatedResults pager={pager}>
+            <table className="tbl">
+              <thead>
+                <tr>
+                  {$c.user_exists
+                    ? <th className="pos"><input type="checkbox" /></th>
+                    : null}
+                  <th>{l('Release')}</th>
+                  <th>{l('Format')}</th>
+                  <th>{l('Tracks')}</th>
+                  <th>{l('Date')}</th>
+                  <th>{l('Country')}</th>
+                  <th>{l('Label')}</th>
+                  <th>{l('Catalog#')}</th>
+                  <th>{l('Barcode')}</th>
+                  {$c.session && $c.session.tport
+                    ? <th>{l('Tagger')}</th> : null}
+                </tr>
+              </thead>
+              <tbody>
+                {releases.map(buildReleaseStatusTable)}
```

Comment:
I would do
```suggestion
                {releases.map(r => buildReleaseStatusTable($c, r))}
```
here and remove all the `CatalystContext` stuff from `buildReleaseStatusTable`.